react-native-libraries.json

```diff
@@ -9053,5 +9053,13 @@
     "android": true,
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/ArunGovil/react-native-flatboard",
+    "npmPkg": "react-native-flatboard",
```

Comment:
```suggestion
```

This field can be skipped, since package name matches the GitHub repository.

Reply:
Thank you @Simek for clarification. npmPkg field has been removed.